packages/perseus/src/widgets/sorter/sorter.tsx

```diff
@@ -103,6 +103,10 @@ class Sorter extends React.Component<Props, State> implements Widget {
     };
 
     render(): React.ReactNode {
+        // TODO(LEMS-2841): Remove client-side shuffle once receiving public
+        //     options. The correct field will already be shuffled.
+        //     Probably easiest to replace "options" with "this.props.correct" when
+        //     setting up Sortable below. Or use the assignment to rename for clarity.
```

Comment:
I'm trying to think through how the sequencing of these changes will work. We might end up in a situation where a single version of Perseus needs to be able to handle both shuffled and unshuffled answers here. In order to distinguish between the shuffled and unshuffled cases, we'd need to include some additional information in the public options, like an `isCorrectShuffled` boolean.

The alternative is to coordinate the release of Perseus that removes the clientside shuffling with the change to webapp that starts consuming the public widget options. I think that kind of coordination limits our maneuverability and can get expensive, so I'd lean toward preferring the boolean. But I could be convinced otherwise.

Reply:
I like the idea of adding a field to be explicit about the state of the data (shuffled or not). That in itself doesn't reveal any information really.

Reply:
I can add that. I'd just be adding it as a field in the public widget options type and then setting it to true in the get function. I think I would need to update the original widget options in the data-scheme.ts file if we want it there all the time, but I'm guess we can check for it and if it's true do something and if it's not there / false do something else.